### DISCUSSION TITLE
Billing

### DISCUSSION THREAD

**Discussion by esaramago** [supabase/supabase#20536] [Questions] [answered]
How many projects can I have with the PRO version without paying more than 25$?
I don't see compute costs anymore in the pricing page.

**Comment by GaryAustin1** [comment, marked as answer]
Every additional project in the org will have a minimum $10 a month for compute.
![image](https://github.com/supabase/supabase/assets/54564956/8a2a98d6-e4c2-4d02-9f37-fafd54f55ea4)

**Reply by esaramago** [reply]
That means that, for example, if I have 3 websites with 3 environments each, I have to pay 25$ + 90$ /month?

**Reply by GaryAustin1** [reply]
Well, $80 as your first one is included.  I am not an employee so just giving you where the info is.  If you have 9 instances then the extra 8 will be $80.  If you have dev instances you might consider local dev for those.

**Reply by esaramago** [reply]
Thanks!
